templates/ts/ai-assistant-bot/teamsapp.local.yml.tpl

```diff
@@ -15,15 +15,19 @@ provision:
       teamsAppId: TEAMS_APP_ID
 
   # Create or reuse an existing Microsoft Entra application for bot.
-  - uses: botAadApp/create
+  - uses: aadApp/create
```

Comment:
minor: will it be confusing that it is "MS Entra app" in description while "aadApp" in action name

Reply:
Just like other tooling (e.g. Azure CLI), we should not rename the action to avoid breaking changes. The description is updated to match the latest product branding.